doc/internal/release/how-to-release.md

```diff
@@ -125,10 +122,10 @@ On the release day, there are several things to do:
   > - The benchmarks need to complete before announcing the blog posts or before they get cross-posted.
 - **Go back to dev mode on the release branch.**
   > - The version constants across the codebase must be updated to `SNAPSHOT`.
-- **Build k8s Docker images and publish them**
+- **Build k8s Docker images and publish them.**
   > - The docker image for `base`, `lite`, etc are built automatically by DockerHub. The k8s images however are dependent on these images and are required to be built manually.
   > - These images should be built after the `base` image has been built and available on DockerHub.
-  > - To build and publish these images, run `./release.sh` from the directory `vitess/docker`.
+  > - To build and publish these images, checkout to the new `git tag` and run `./release.sh` from the directory `vitess/docker`.
```

Comment:
```suggestion
  > - To build and publish these images, checkout the new `git tag` and run `./release.sh` from the directory `vitess/docker`.
```

Reply:
`git tag` looks like a command. Maybe you mean something like:

- To build and publish these images, checkout the new release tag that was just created and run `./release.sh` from the directory `vitess/docker`.

Reply:
Yes that's what I mean, i'll change it

Reply:
This is fixed via 4d509fc185